core/spec.md

```diff
@@ -1117,6 +1122,47 @@ HTTP body.
 
 ---
 
+### Registry Well-Known Config
+
+The registry comes with a well known location `/.well-known/xregistry.json`,
+following [RFC8615](https://datatracker.ietf.org/doc/html/rfc8615).
+
+It acts as a single entrypoint from where locations and capabilities of the
+xRegistry implementation can be retrieved. Since this URL is always the same,
+it can be used to find out whether xRegistry is supported and where to find the API.
+It also allows to support multiple versions of the xRegistry protocol at the same time
+and allows implementers to choose their own API base path.
+
+```yaml
+{
+  # The following top-level attributes are identical to the xRegistry API root response
+  # We may consider removing them here if we're bothered by the duplication.
```

Comment:
Some of these (like labels) might be container private data, and "self" wouldn't be accurate. It might be best to either remove these or sub-set them down to just simple (safe) things like "id", and make it optional since even that could expose too much info.